network/utils.py

```diff
@@ -8,8 +8,10 @@
 from tensorflow.keras.callbacks import Callback
 from tensorflow.keras.losses import binary_crossentropy
 
+from bayesian_cnn_prometheus.constants import Paths
+
 ex = Experiment()
-ex.add_config("configs/config.json")
+ex.add_config(str(Paths.CONFIG_PATH.resolve()))
```

Comment:
Jeśli wyrzucamy sacred to to powinno też być do wyrzucenia nie?

Reply:
To jest w network. Zmieniłem, to bo przy odpalaniu chciało inny typ ale nie wiem, czy można to bezpiecznie usunąć. Musiałbym wytestować na danych i sprawdzić czy coś się dzieje bez tego.

Ale poprostu o ile nie trzeba było, to modułu network nie ruszałem